pom.xml

```diff
@@ -25,7 +25,7 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <liquibase.version>4.29.0</liquibase.version>
+        <liquibase.version>DAT-18196-SNAPSHOT</liquibase.version>
```

Comment:
@StevenMassaro do we need to change it back before merging into 'main'?

Reply:
If we do that, artifacts won't be published to the draft release and the code won't compile.

That being said, https://datical.atlassian.net/browse/DAT-18488 should fix this.